lang/cs/Org.Apache.REEF.Client.Tests/JobResourceUploaderTests.cs

```diff
@@ -64,11 +69,19 @@ public void UploadJobResourceReturnsJobResourceDetails()
             var testContext = new TestContext();
             var jobResourceUploader = testContext.GetJobResourceUploader();
 
-            var jobResource = jobResourceUploader.UploadJobResource(AnyDriverLocalFolderPath, AnyDriverResourceUploadPath);
+            var archiveJobResource = jobResourceUploader.UploadArchiveResource(AnyDriverLocalFolderPath, AnyDriverResourceUploadPath);
+            var fileJobResource = jobResourceUploader.UploadFileResource(AnyLocalJobFilePath, AnyDriverResourceUploadPath);
+            var jobResources = new List<JobResource> { archiveJobResource, fileJobResource };
```

Comment:
Separate tests for the two API calls? Makes is much easier to isolate mistakes and debug.